### PR TITLE
Fix issue on angle tolerance in computeHullThickness

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Fix issue [#1629](https://github.com/DGtal-team/DGtal/issues/1629) on computeHullThickness by adding angle tolerance to detect co-linearity vectors. Thanks @aramadancs!

# Checklist

- [ ] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [ ] Doxygen documentation of the code completed (classes, methods, types, members...)
- [ ] Documentation module page added or updated.
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [ ] No warning raised in Debug mode.
- [ ] All continuous integration tests pass (Github Actions & appveyor)

### CODE REVIEW

File: src/DGtal/geometry/tools/Hull2DHelpers.ih [hunk 1]
```diff
@@ -361,8 +362,8 @@ namespace DGtal
             }
           return 0.0;
           }
-        
-        while(getAngle(aConvexHull[i], aConvexHull[i+1], aConvexHull[j], aConvexHull[j+1]) < M_PI ){
+        while(getAngle(aConvexHull[i], aConvexHull[i+1],
+                       aConvexHull[j], aConvexHull[j+1]) < M_PI - angleTolerance ){
```
Comment (dcoeurjo) [inline src/DGtal/geometry/tools/Hull2DHelpers.ih:366]: why not use a exact geometrical predicate for convex hull computation? If I remember correctly, @troussil has got a collection of 2D predicates already available in DGtal

Comment (kerautret) [reply src/DGtal/geometry/tools/Hull2DHelpers.ih:366]: Mainly for the use for alpha-thick segment, If I well remember.

Comment (dcoeurjo) [reply src/DGtal/geometry/tools/Hull2DHelpers.ih:366]: Shouldn't we fix this cvx computation with exact computations and not angle comparisons ? 
(for 1.3 or maybe the next one)

Comment (kerautret) [reply src/DGtal/geometry/tools/Hull2DHelpers.ih:366]: for non integer input the tolerance is probably useful, else perhaps waiting after 1.3 could let more time

Comment (dcoeurjo) [reply src/DGtal/geometry/tools/Hull2DHelpers.ih:366]: ok could you please add an issue on this? Just to remember that this should be fixed.

**Review by dcoeurjo — APPROVED** [review]

File: src/DGtal/geometry/tools/Hull2DHelpers.ih (hunk 1 above)
Comment (kerautret) [reply src/DGtal/geometry/tools/Hull2DHelpers.ih:366]: yes I add it

**Pull request merged**